### DISCUSSION TITLE
Argument list too long to when loading an eBPF program, Why is this?

### DISCUSSION THREAD

**Discussion by Mr-XGJ** [apache/skywalking#12261] [Q&A]
log: executing profiling task failure, taskId: 75bb766b0a680f013b5df08e6c3d51de514afff438c2056cc7e0871409039d98, reason: field SysReadvRet: program sys_readv_ret: load program: argument list too long: 882: (79) r7 = *(u64 *)(r6 +24): R0_w=invP(id=0) R6=map_value(id=0,o (truncated, 597 line(s) omitted)

**Comment by mrproliu** [comment]
Please provide more information about the version:
1. System kernel version
2. Rover version

**Reply by Mr-XGJ** [reply]
1. kernel 5.10
2. Rover v0.5.0

**Reply by mrproliu** [reply]
Your kernel version seems to meet the requirements. Have you tried testing with a more higher-level kernel? For example, 5.15. Currently, due to the log being truncated, there is no better way to identify the cause of this issue.

**Reply by Mr-XGJ** [reply]
Thank you so much for your suggestion.
